content/intro-to-storybook/react/en/deploy.md

```diff
@@ -1,91 +1,162 @@
 
-## Continuous deployment
+## Publish Storybook
 
-We want to share the latest version of components whenever we push code. To do this we need to continuously deploy Storybook. We’ll rely on GitHub and Netlify to deploy our static site. We’re using the Netlify free plan.
+In this tutorial we use <a href="https://www.chromatic.com/">Chromatic</a>, a free publishing service made by the Storybook maintainers. This allows us to deploy and host our Storybook safely and securely in the cloud.
 
-### GitHub
+### Setup a repository in GitHub
 
-First you want to setup Git for your project in the local directory. If you're following along from the previous testing chapter jump to setting up a repository on GitHub.
+Before we begin, our local code needs to sync with a remote version control service. When our project was initialized in the [Get started chapter](/react/en/get-started/), Create React App (CRA) already created a local repository for us. At this stage it's safe to add our files to the first commit.
 
-When the project was initialized with Create React App, a local repository was already setup for you. At this stage it's safe to add the files to the first commit.
+Issue the following commands to add and commit the changes we've done so far.
 
 ```bash
 $ git add .
 ```
 
-Now commit the files.
+Followed by:
 
 ```bash
 $ git commit -m "taskbox UI"
 ```
 
-### Setup a repository in GitHub
-
-Go to GitHub and setup a repository [here](https://github.com/new). Name your repo “taskbox”.
+Go to GitHub and create a new repository for our project [here](https://github.com/new). Name the repo “taskbox”, same as our local project.
 
 ![GitHub setup](/intro-to-storybook/github-create-taskbox.png)
 
-In the new repo setup copy the origin URL of the repo and add it to your git project with this command:
+In the new repo, grab the origin URL of the repo and add it to your git project with this command:
 
 ```bash
 $ git remote add origin https://github.com/<your username>/taskbox.git
 ```
 
-Finally push the repo to GitHub
+Finally, push our local repo to the remote repo on GitHub with:
 
 ```bash
 $ git push -u origin master
 ```
 
-### Netlify
+### Get Chromatic
+
+Add the package as a dependency.
+
+```bash
+yarn add -D chromatic
+```
+
+Once the package is installed, [login to Chromatic](https://www.chromatic.com/start) with your GitHub account (Chromatic will only ask for lightweight permissions). Then we'll create a new project called name "taskbox" and synch it with the GithHub repository we've setup.
+
+Click `Choose GitHub repo` under collaborators and select your repo.
+
+And don't forget to copy the unique `project-token` that was generated.
+
+<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
+    type="video/mp4"
+  />
+</video>
```

Comment:
@domyen like we've discussed can you make a new animation for this? To address the documentation flow?

Reply:
Done!